app/services/import_companies.rb

```diff
@@ -76,6 +97,12 @@ def get_questions(row)
     end.compact
   end
 
+  def get_emissions(row)
+    row.headers.grep(/^\d{4}$/).map do |year|
```

Comment:
what do you think about reformatting this like this?
```rb
    row.headers
      .grep(/^\d{4}$/)
      .map { |year| { year.to_s.to_i => row[year]&.to_f } }
      .reduce(&:merge)
      .reject { |_k, v| v.blank? }
```

Reply:
right :+1: I'm not taking much care into those importers, not sure if they will stay